### DISCUSSION TITLE
My Modded Minecraft 1.16.5 wont launch

### DISCUSSION THREAD

**Discussion by masonbooh** [FabricMC/fabric-api#1442] [Player support] [answered]
Could not find required mod: enva requires {fabricloader @ [>=0.11.3]}
Creating Empty clause ?

everytime i try to launch it gives me this massage

i need help let me know if you need my mod list to figure out the problem

**Comment by haykam821** [comment, marked as answer]
Install a newer version of the Fabric loader. You should be able to install a newer version using [this tutorial](https://fabricmc.net/wiki/install).

**Reply by masonbooh** [reply]
it gave me a file called intermediary-fabric-loader and i dont know what to do from here

**Reply by masonbooh** [reply]
ima try to run it real quick

also sorry for seeing this late i was watching youtube while waiting for a response

**Reply by masonbooh** [reply]
now its saying

Could not find required mod: enva requires {fabric @ [*]}
Creating Empty clause ?

**Reply by masonbooh** [reply]
ima just look for a video tutorial

thanks for tryin to help me

**Reply by masonbooh** [reply]
turns out it was some of the mods i used that broke it

the mods that caused this problem ar haema and moborigins

sorry for the ruckus
